docs/Release_notes.md

```diff
@@ -1,6 +1,28 @@
 CARMA Platform Release Notes
 ----------------------------
 
+Version 3.3.0, released TBD
+--------------------------------------------------------
+
+**Summary:**
+CARMAPlatform release version 3.3.0 is comprised of two key enhancements. Firstly the plugin manager is now able to return a list of services that match a user-requested capability. Second, there are new configurations for the Freightliner Cascadia, including lidar configurations which allow autoware to combine the two lidar devices' data into a single stream. 
+
+Enhancements in this release:
+- Issue 478: Capability Interface Implementation
+- Issue 489: Create configuration of carma-platform capable of operating on the freightliner cascadia using two lidars
+
+Fixes in this release:
+- Issue 488: CARMAGarminLidarLiteV3DriverWrapper does not have functional docker image
+- Issue 490: libproj.so can not be found by vector_map loader
+- Issue 496: Update Lanlet2 to version 0.9.0
+- Issue 497: PACMOD and/or SSC module status messages appear to have different semantics on truck
+- Issue 498: Velodyne Driver Registration Not Functional on Truck
+- Issue 499: Trailer angle sensor devices are not mapped on the host or in docker
```

Comment:
PLEASE check with John if we need to put a note on the existing hardware issue with the red truck that Ash is working on and have open ticket with AStuff.

Reply:
@TonysCousin thoughts on this?

Reply:
@maefromm I spoke with John and he says that we don't want to include hardware issues in the release notes